.circleci/config.yml

```diff
@@ -146,18 +146,31 @@ jobs:
           name: Scala Samples
           command: |
             if [ $(find ./samples -name plugins.sbt | xargs grep -h "akkaserverless-sdk.version" | uniq | wc -l) -ne 1 ]; then
-              "Not all Scala samples have the same SDK version:"
+              echo "Not all Scala samples have the same SDK version:"
               find ./samples -name plugins.sbt | xargs grep "akkaserverless-sdk.version"
               false
             fi
       - run:
           name: Java Samples
           command: |
             if [ $(find ./samples -name pom.xml | xargs grep -h "<akkaserverless-sdk.version>" | tr -s [:space:] | uniq | wc -l) -ne 1 ]; then
-              "Not all Java samples have the same SDK version:"
+              echo "Not all Java samples have the same SDK version:"
               find ./samples -name pom.xml | xargs grep "<akkaserverless-sdk.version>"
               false
             fi
+      - run:
+          name: Protoc version
+          command: |
+            AKKA_GRPC_PROTOC=$(sbt "sdkCore/protocVersion" 2>/dev/null | tail -n 1 | sed 's/\[info\] //g')
+            POM_PROTOC_VERSIONS=$(find ./samples ./maven-java/ -name pom.xml | xargs egrep -h "com.google.protobuf:protoc:" | sed 's/.*:protoc://' | sed 's/:exe.*//' | sort | uniq)
```

Comment:
neato

Reply:
were you trying to write my name or to say 'neat'?

Reply:
https://www.urbandictionary.com/define.php?term=neato - though perhaps `|) A |\/| |\|` was onto something...

Reply:
Lol. New nick name decided!